src/components/label/label.e2e.ts

```diff
@@ -2,7 +2,7 @@ import { newE2EPage } from "@stencil/core/testing";
 import { renders, hidden } from "../../tests/commonTests";
 
 describe("calcite-label", () => {
-  it("renders", () => renders("calcite-label", { display: "inline" }));
+  it("renders", () => renders("calcite-label", { display: "flex" }));
```

Comment:
why are we testing the existence of CSS props here? Shouldn't that be covered by the screenshot tests?

Reply:
We're testing the default display of components because that may not necessary change a screenshot and we don't want to break changes for users.